integration-test/1425-more-osm-features.py

```diff
@@ -0,0 +1,1377 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class MoreOSMFeaturesTest(FixtureTest):
+
+    def test_sand_beach_way(self):
+        import dsl
+
+        z, x, y = (16, 19335, 24602)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/108403305
+            dsl.way(108403305, dsl.tile_box(z, x, y), {
+                'ele': u'2',
+                'gnis:feature_id': u'959437',
+                'name': u'Orchard Beach',
+                'natural': u'beach',
+                'source': u'openstreetmap.org',
+                'surface': u'sand',
+                'wikidata': u'Q7100176',
+                'wikipedia': u'en:Orchard Beach, Bronx',
+            }),
+        )
+
+        for layer in ('pois', 'landuse'):
+            self.assert_has_feature(
+                z, x, y, layer, {
+                    'id': 108403305,
+                    'kind': u'beach',
+                    'kind_detail': u'sand',
+                })
+
+    def test_pebblestone_beach_way(self):
+        import dsl
+
+        z, x, y = (16, 19253, 24394)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/233695433
+            dsl.way(233695433, dsl.tile_box(z, x, y), {
+                'access': u'yes',
+                'name': u'Lake Minnewaska Beach',
+                'natural': u'beach',
+                'source': u'openstreetmap.org',
+                'surface': u'pebblestone',
+            }),
+        )
+
+        for layer in ('pois', 'landuse'):
+            self.assert_has_feature(
+                z, x, y, layer, {
+                    'id': 233695433,
+                    'kind': u'beach',
+                    'kind_detail': u'pebblestone',
+                })
+
+    def test_gravel_beach_way(self):
+        import dsl
+
+        z, x, y = (16, 19304, 24371)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/way/360547782
+            dsl.way(360547782, dsl.tile_box(z, x, y), {
+                'natural': u'beach',
+                'source': u'openstreetmap.org',
+                'surface': u'gravel',
+                # fake name to get the POI to appear
```

Comment:
Nice ;)

Reply:
This is true for pois, but for landuse the polygon would still show up regardless of name being present?

Reply:
Yup, the fake name is only needed for POIs. I've split the test in 4f20bd3 between `pois` and `landuse`, with the fake name only on the `pois` test.